cmake/external_mkldnn.cmake

```diff
@@ -48,6 +48,7 @@ set(MKLVERSION "2018.0.3.20180406")
 if (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
     set(MKLPACKAGE "mklml_lnx_${MKLVERSION}.tgz")
     set(MKL_SHA1_HASH aea0d9ce65773cfcf5d8292b8db553bde965fc8f)
+    set(MKL_LIBS libiomp5.so libmklml_intel.so)
```

Comment:
I'm guessing a similar line will be needed for `Darwin` but we can deal with that later.

Reply:
yep

Reply:
and windows too